web/src/components/atomic/Icon.vue

```diff
@@ -7,7 +8,7 @@
   <i-mdi-gesture-tap v-else-if="name === 'manual-pipeline'" class="h-6 w-6" />
   <i-mdi-tag-outline v-else-if="name === 'tag'" class="h-6 w-6" />
   <i-clarity-deploy-line v-else-if="name === 'deployment'" class="h-6 w-6" />
-  <i-mdisource-commit v-else-if="name === 'commit'" class="h-6 w-6" />
+  <i-mdi-source-commit v-else-if="name === 'commit'" class="h-6 w-6" />
```

Comment:
was this a bug? does that icon exist?

Reply:
I think both work, with the `-` even the inline preview of my VSCode plugin works, but not tested in the app.